test/e2e/cypress/support/commands.js

```diff
@@ -37,6 +37,46 @@ Cypress.Commands.add("AuthLogin", () => {
    });
 });
 
+Cypress.Commands.add("ModelCreate", (ID, data) => {
+   cy.window().then((win) => {
+      let AB = win.AB;
```

Comment:
So window.AB to my understanding is meant to be removed at some point, or at least there is a comment suggesting that in [`ab_platform_web`](https://github.com/digi-serve/ab_platform_web/blob/f7f66381c8b1b4f89550aea8724fe7d09758454a/init/Bootstrap.js#L211) 
Is there another why we can get the facory? Or should we add a comment, to `ab_paltform_web` warning that our tests expect this?

Reply:
Yeah, unless we can think of another way to access the Factory without doing this, we should probably update the `ab_platform_web` and make note that our tests are expecting this.  

I can go ahead and do that.